pkg/shell/superuser.jsx

```diff
@@ -288,9 +288,9 @@ export class SuperuserIndicator extends React.Component {
     render() {
         function create_trigger(unlocked, onclick) {
             return (
-                <span className="navbar-text" onClick={onclick}>
+                <a className="navbar-text" onClick={onclick}>
```

Comment:
Should we actually turn it into a `Button`? at least `variant="link"`, but maybe even "secondary" so that it looks more clickable?